improver/wxcode/weather_symbols.py

```diff
@@ -105,6 +105,29 @@ def __repr__(self):
         return '<WeatherSymbols tree={} start_node={}>'.format(self.wxtree,
                                                                self.start_node)
 
+    @staticmethod
+    def _is_variable(thing):
```

Comment:
I have fixed the CLI help string. I hadn't realised it gave this output, or that there was no unit-test covering it (there still isn't). I'm reluctant to specify that "probaility_of_" must be in all the parameter names as it makes it more difficult to change the weather tree. It is conceivable that a percentile (e.g. median) or deterministic value could be wanted to inform a decision node.

Reply:
I guess before it probably wouldn't have been picked up by a help CLI test, which I don't think we have any more since moving to pytest for CLI tsets. Do we need to add unit tests for this? I'm not sure they would have helped catch it in this case.

Reply:
I have added a pytest which would catch this error in the future.